hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -271,43 +288,53 @@ async def _place_order(
         base = tokens[0]
         quote = tokens[1]
 
-        # this is temporarily hard coded to a single solana wallet
-        base_addr = CONSTANTS.TOKEN_PAIR_TO_WALLET_ADDR[base]
-        quote_addr = CONSTANTS.TOKEN_PAIR_TO_WALLET_ADDR[quote]
-        payer_address = base_addr if side == api.Side.S_ASK else quote_addr
+        payer_address = self._token_accounts[base] if side == api.Side.S_ASK else self._token_accounts[quote]
+
+        open_orders_address = ""
+        if trading_pair in self._open_orders_addresses:
+            open_orders_address = self._open_orders_addresses[trading_pair]
 
         blxr_client_order_id = convert_hummingbot_to_blxr_client_order_id(order_id)
         self._hummingbot_to_solana_id[order_id] = blxr_client_order_id
 
-        submit_order_response = await self._provider_1.submit_order(
+        post_order_response = await self._provider_1.post_order(
             owner_address=self._sol_wallet_public_key,
             payer_address=payer_address,
             market=trading_pair,
             side=side,
-            types=[type],
+            type=[type],
             amount=float(amount),
             price=float(price),
+            open_orders_address=open_orders_address,
+            client_order_i_d=blxr_client_order_id,
             project=OPENBOOK_PROJECT,
-            client_order_id=blxr_client_order_id,
-            open_orders_address=self._open_orders_address,
+        )
+
+        signed_tx = signing.sign_tx(post_order_response.transaction.content)
```

Comment:
any reason you want to split the signing here and not use the submit methods in the Python SDK?